### PR TITLE
Stats page kevin

### PR DESCRIPTION
**Pull request by KevinAlamo**
I created the Stats Page servlet and added Datastore logic which will count the number of messages on the site. Some of the commits fixed mistakes I made in where I put the code.

### CODE REVIEW

**Comment by KevinAlamo** [comment]
This branch was done first

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -79,4 +80,10 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  
+  public int getTotalMessageCount(){
+		 Query query = new Query("Message");
+		 PreparedQuery results = datastore.prepare(query);
+		 return results.countEntities(FetchOptions.Builder.withLimit(1000));
+	 }
```
Comment (matthew-chinn) [inline src/main/java/com/google/codeu/data/Datastore.java:118]: we'll work on getting a formatter at the next meeting, but for now can we use 2 spaces of indentation like the rest of the Java code?

**Review by matthew-chinn — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (KevinAlamo) [reply src/main/java/com/google/codeu/data/Datastore.java:118]: I added the spaces

**Review by matthew-chinn — APPROVED** [review]

**Comment by matthew-chinn** [comment]
Ooh looks like you have conflicts! Exciting :)

**Pull request merged**